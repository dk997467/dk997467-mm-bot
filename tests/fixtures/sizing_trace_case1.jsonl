{"ts":"2025-01-01T00:00:00Z","symbol":"BTCUSDT","current":0.0,"desired":10.0,"breach":false}
{"ts":"2025-01-01T00:00:10Z","symbol":"BTCUSDT","current":10.0,"desired":25.0,"breach":false}
{"ts":"2025-01-01T00:00:20Z","symbol":"BTCUSDT","current":25.0,"desired":60.0,"breach":true}
{"ts":"2025-01-01T00:00:30Z","symbol":"BTCUSDT","current":40.0,"desired":90.0,"breach":true}
{"ts":"2025-01-01T00:00:40Z","symbol":"BTCUSDT","current":55.0,"desired":95.0,"breach":true}
{"ts":"2025-01-01T00:00:50Z","symbol":"BTCUSDT","current":70.0,"desired":80.0,"breach":false}
{"ts":"2025-01-01T00:01:00Z","symbol":"BTCUSDT","current":75.0,"desired":74.0,"breach":false}
{"ts":"2025-01-01T00:01:10Z","symbol":"BTCUSDT","current":74.0,"desired":74.0,"breach":false}
{"ts":"2025-01-01T00:01:20Z","symbol":"BTCUSDT","current":74.0,"desired":72.0,"breach":false}
{"ts":"2025-01-01T00:01:30Z","symbol":"BTCUSDT","current":72.0,"desired":72.0,"breach":false}

