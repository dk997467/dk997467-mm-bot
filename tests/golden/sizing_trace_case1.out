2025-01-01T00:00:00Z BTCUSDT 0.000000 10.000000 10.000000 0.000000 0 0.000000 0.000000
2025-01-01T00:00:10Z BTCUSDT 10.000000 25.000000 15.000000 1.500000 0 1.500000 11.500000
2025-01-01T00:00:20Z BTCUSDT 25.000000 60.000000 35.000000 3.750000 1 3.750000 28.750000
2025-01-01T00:00:30Z BTCUSDT 40.000000 90.000000 50.000000 4.000000 2 3.500000 43.500000
2025-01-01T00:00:40Z BTCUSDT 55.000000 95.000000 40.000000 4.125000 2 2.750000 57.750000
2025-01-01T00:00:50Z BTCUSDT 70.000000 80.000000 10.000000 10.500000 2 5.250000 75.250000
2025-01-01T00:01:00Z BTCUSDT 75.000000 74.000000 -1.000000 11.250000 2 -0.500000 74.500000
2025-01-01T00:01:10Z BTCUSDT 74.000000 74.000000 0.000000 11.100000 2 0.000000 74.000000
2025-01-01T00:01:20Z BTCUSDT 74.000000 72.000000 -2.000000 11.100000 2 -1.000000 73.000000
2025-01-01T00:01:30Z BTCUSDT 72.000000 72.000000 0.000000 10.800000 2 0.000000 72.000000

